app/src/main/java/com/nervousfish/nervousfish/modules/IModule.java

```diff
@@ -0,0 +1,8 @@
+package com.nervousfish.nervousfish.modules;
+
+/**
+ * Defines a module that can be used by a service locator.
```

Comment:
You can use `@see com.nervousfish.nervousfish.ServiceLocator` to reference the class 😉